Realm.Shared/Dynamic/DynamicRealmObjectHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Realms.Weaving;
+
+namespace Realms.Dynamic
+{
+    internal class DynamicRealmObjectHelper : IRealmObjectHelper
```

Comment:
This object is so small and inexpensive to create that `Lazy`-fying it would be overkill, IMO.